doom-themes-ext-treemacs.el

```diff
@@ -130,8 +144,8 @@ This is used to generate extensions for `treemacs' from `all-the-icons-icon-alis
 
   ;; Disable fringes (and reset them everytime treemacs is selected because it
   ;; may change due to outside factors)
-  (add-hook 'treemacs-mode-hook #'doom-themes-hide-fringes)
-  (advice-add #'treemacs-select-window :after #'doom-themes-hide-fringes)
+  (add-hook 'treemacs-mode-hook #'doom-themes-hide-fringes-maybe)
+  (advice-add #'treemacs-select-window :after #'doom-themes-hide-fringes-maybe)
```

Comment:
This is needed for the doom module
`(advice-add #'+treemacs/toggle :after #'doom-themes-hide-fringes-maybe)` ?

Reply:
It isn't, no. `+treemacs/toggle` still uses `treemacs`/`treemacs-add-and-display-current-project`, which both use `treemacs-select-window`.